Resources/doc/entity-listeners.rst

```diff
@@ -15,11 +15,7 @@ entity manager it should be registered with. Example:
                 class: \UserListener
                 tags:
                     - { name: doctrine.orm.entity_listener }
--
-                        name: doctrine.orm.entity_listener
-                        event: preUpdate
-                        entity: App\Entity\User
-                        entity_manager: custom
+
```

Comment:
This looks wrong, and seems related to #897

Reply:
Indeed :+1:

Reply:
#897 was merged. Please rebase to remove the change here.

Reply:
Rebased. Keeping the new empty line is ok? Improves readability when editing it imho